julia.html.markdown

```diff
@@ -192,8 +192,8 @@ sort!(arr) # => [4,5,6]; arr is now [4,5,6]
 
 # Looking out of bounds is a BoundsError
 try
-    a[0] # => ERROR: BoundsError() in getindex at array.jl:270
-    a[end+1] # => ERROR: BoundsError() in getindex at array.jl:270
+    a[0] # => BoundsError([7,2,4,3,4,5,6],(0,))
+    a[end+1] 
```

Comment:
In Juno the error appears only for the first error...probably after the first exception happens the code goes directly to the catch block